### ISSUE TITLE
'ClinCheck Video' functionality

### ISSUE THREAD

**Issue by OlenkaP** [RiLights/aligntime#42]
1.Change name from "Upload ClinCheck Video' to 'Clincheck Video'

2. This functionality should allow users to upload ClinCheck video treatment plan provided by the orthodontist to store it in the AlignTime App so that they can view it at any time needed.

3. Users should be able to upload multiple videos (they are normally only up to a few minutes long) and store them locally in the AlignTime.

4. Users should be able to remove those videos from AlignTime, should they decide to do so 

![image](https://user-images.githubusercontent.com/52235757/72690592-9fbd6c00-3b82-11ea-9522-c3d2b189fcd5.png)

**Comment by RiLights** [comment]
Upload Video?
Do you mean real video?
Can you please send an example of this video?

**Comment by OlenkaP** [comment]
Yeah Orthodontist provides users with treatment plan in week-by-week basis, video is based on 3d model of the teeth and prognosis of how they gonna move.

I would share it privately with you

**Comment by RiLights** [comment]
Should we embed video player in AlignTime?

**Comment by OlenkaP** [comment]
would we be able to just use one of the standard tools to play video through AlignTime? what's involved into embedding Video Player?
I think we need the easiest to implement solution on order to let users view video through aligntime

**Comment by RiLights** [comment]
Need to investigate this area.

Why user required this video?

**Comment by OlenkaP** [comment]
It's nice to see, what was the plan for your teeth to look on let's say week 24 and you can see it on that video 
plus it has entire overview of how teeth would look like during the treatment

**Comment by OlenkaP** [comment]
so that's why users would need that video 

i.e. i keep mine just in the email and look into it every few weeks, but i would love to have it in one place (aligners reminder app)

**Comment by RiLights** [comment]
I'm wondering, why Orthodontist doesn't provide all videos at once? Why it can't be just one video (from start to end)?

**Comment by OlenkaP** [comment]
so they provide video from start till the end at once, but sometimes treatment plan can change, therefore video would change and users would have a new video
